tools/topology/topology1/sof-tgl-max98357a-rt5682.m4

```diff
@@ -248,7 +248,7 @@ DAI_ADD(sof/pipe-dai-capture.m4,
 `
 ifdef(`2CH_2WAY',`# No echo reference for 2-way speakers',
 `# currently this dai is here as "virtual" capture backend
-W_DAI_IN(SSP, SPK_SSP_INDEX, SPK_SSP_NAME, FMT, 3, 0, 0)
+W_DAI_IN(SSP, SPK_SSP_INDEX, SPK_SSP_NAME, FMT, 3, 0, 0, 2)
```

Comment:
@aiChaoSONG are you sure you want to use pipeline ID 2 for the virtual DAI capture? It is already in use for SSP0 playback isnt it?

Reply:
@ranj063 you mean we should use a pipeline ID which is not taken?